docs/api/config.json

```diff
@@ -17,7 +17,7 @@
         },
         {
             "typeMatchPrefix": "^k8s\\.io/(api|apimachinery/pkg/apis)/",
-            "docsURLTemplate": "https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
+            "docsURLTemplate": "https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
```

Comment:
We should use the same default version as the operator, `v1.31`, which can be found in `./tools/test.env`.

Reply:
thanks! Will Fix!

Reply:
v1.32 since https://github.com/planetscale/vitess-operator/commit/36dd055118d101c15724a8b314cc8e552dd47e2a

Reply:
fixed!